src/app/docs/tooltip.component.ts

```diff
@@ -7,11 +7,16 @@ import { Analytics } from './shared/analytics';
     <docs-wrapper component="Tooltip">
         <h3>Demo</h3>
         <demo-tooltip></demo-tooltip>
+        <br>
+        <br>
+        <h3>Usage</h3>
```

Comment:
I think you could use something like `class="mt-3"` to add space above this heading, instead of `<br>`s.
Not to be worried though, just a comment :)